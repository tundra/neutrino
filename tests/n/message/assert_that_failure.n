# Copyright 2014 the Neutrino authors (see AUTHORS).
# Licensed under the Apache License, Version 2.0 (see LICENSE).

import $assert;
do $assert:that(false);
