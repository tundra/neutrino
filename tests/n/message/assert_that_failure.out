--- backtrace ---
- leave.assert_failed()
- :fail()
- :that(false)
Error: %<condition: Signal(escape)>
